Womens-Hackathons-and-Events.md

```diff
@@ -2,21 +2,28 @@
 
 Crowdsourced list of all women's hackathons and other events promoting women in tech. Contributions welcome! Create a PR or contact [@alainakafkes](https://github.com/alainakafkes) to update.
 
-## January
+## Oct 2016
 
-## February
+## Nov 2016
+* [Lady Problems Hackathon - Boston](https://www.eventbrite.com/e/lady-problems-hackathon-boston-tickets-27716935099) 
```

Comment:
Lady Problems Hackathon is already included in our list. This commit introduces formatting issues to the document.